util/install.sh

```diff
@@ -180,7 +180,7 @@ function mn_deps {
 
     echo "Installing Mininet core"
     pushd $MININET_DIR/mininet
-    sudo PYTHON=${PYTHON} make install
+    sudo PYTHON=${PYTHON} make PYTHON=${PYTHON} install
```

Comment:
So I just tested this, and it appears that 
    sudo PYTHON=${PYTHON} make install
is fine. Do your tests indicate otherwise?

Reply:
Hi,

To check this and faucetsdn pull request, I have made a 
```util/install.sh -3nvfy```
on a clean Ubuntu 20.04 VM with the Mate desktop and I have a running mininet with an updated Ryu controller

Best, /PA